### ISSUE TITLE
Add ability to install package dependencies from package.xml, for CI workflows

### ISSUE THREAD

**Issue by alan-copeland-keysight** [opentap/opentap#1039]
It would be useful to be able to install package.xml dependencies during a CI build by passing 1) location of package.xml 2) flag like --dependencies-only.

This would allow the dependencies to be installed easily by CI system, and dependencies would only have to be specified in a single location.

**Comment by alan-copeland-keysight** [comment]
Should be an enhancement request, not a bug _(I don't have permissions to set labels)_

**Comment by alnlarsen** [comment]
@alan-copeland-keysight I'm assuming you need this because you get an error during `tap package create` if the package dependencies are not installed? If that's the case, maybe we need to add the flag `--install-missing-dependencies` to `tap package create`?

**Comment by alan-copeland-keysight** [comment]
Yes, you nailed the usecase. **--install-missing-dependencies** sounds like it would resolve the issue. 

Should that flag be the default? Or is there a case where you would want to **tap package create**, but not refresh the dependencies first?

**Comment by alnlarsen** [comment]
We always prefer not to alter previous behavior when possible. Even if I can't think of a scenario where you would prefer `package create` to fail, it could impact someone in unexpected ways.
